src/global_mapper_ros/global_mapper_ros.h

```diff
@@ -3,9 +3,11 @@
 
 #include <csignal>
 #include <memory>
+#include <string>
+#include <vector>
 
 #include "ros/ros.h"
-#include <tf2_ros/transform_listener.h>
+#include "tf2_ros/transform_listener.h"
```

Comment:
The brackets force the compiler to look in the system includes first.  The quotes let the compiler follow the paths as determined by your CMakeLists.  Given this, I only include std library stuff with brackets.

"The #include <filename> is used when a system file is being referred to. ... The simple general rule is to use angled brackets to include header files that come with the compiler. Use double quotes to include any other header files. Most compilers do it this way"

http://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename